Bandaid fix OperatorHub tests for etcd operator

SamiSousa opened the pull request:
- Changed OperatorHub test to check for OwnNamespace installmode
  rather than AllNamespaces
- Updated namespace to "default" to match the new install namespace

The etcd operator in community-operators changed late last week,
replacing AllNamespaces functionality with SingleNamespace.
This puts a bandaid on the issue.
Operators in community-operators can change frequently, and
because of this OperatorHub tests will be unstable when relying on
external content.

/cc @galletti94 @jeff-phillips-18 @spadgett @dtaylor113 

This will help tests pass for now, but it's not a long-term solution. I suggest we either use a separate quay namespace with stable operator bundles and add that as a "Custom" OperatorSource during this test, or possibly figure out installmodes on the fly.

galletti94 commented:
/retest

galletti94 commented:
@spadgett I think this is why all the tests were failing. Thanks @SamiSousa for catching this!

galletti94 commented:
/retest

spadgett commented:
/retest

SamiSousa commented:
/retest

SamiSousa commented:
/retest

SamiSousa commented:
```
operator Build image console from the repository 0.37s
Command: go run hack/e2e.go -v -test --test_args='--ginkgo.focus=operator\sBuild\simage\sconsole\sfrom\sthe\srepository$'

could not wait for build: the build console failed with reason DockerBuildFailed: Docker build strategy has failed.

[94] ./node_modules/cache-loader/dist/cjs.js!./node_mo...ader.js??ref--6-5!./public/vendor.scss 1.06 MiB {0} [built]
        + 94 hidden modules
error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error: build error: running './build.sh' failed with exit code 1
```

spadgett commented:
/retest

spadgett commented:
@SamiSousa OperatorHub tests failed :(

spadgett commented:
/retest

SamiSousa commented:
@spadgett The OperatorHub and OLM test suites passed for me when I created the PR this morning, launching a new cluster to double check now.

SamiSousa commented:
Updated tests to use `testName` as the namespace rather than `default`, and added some views for the Subscription page to help with selecting namespace for the SingleNamespace installMode.

spadgett commented on frontend/integration-tests/views/operator-hub.view.ts:
```diff
@@ -16,6 +16,13 @@ export const createSubscriptionFormTitle = element(by.cssContainingText('h1', 'C
 export const createSubscriptionFormBtn = element(by.buttonText('Subscribe'));
 export const createSubscriptionFormInstallMode = element(by.cssContainingText('label', 'Installation Mode'));
 
+export const installNamespaceDropdown = $('.dropdown--full-width');
```
We might give this a more descriptive ID than we're currently using in the form, and then we can select the right dropdown by ID here.

Don't worry about this if the PR passes tests, though. We need to unblock the queue.

spadgett commented:
Tests are green!
